app/controllers/v0/education_benefits_claims_controller.rb

```diff
@@ -14,7 +14,7 @@ def create
 
       StatsD.increment("#{stats_key}.success")
       Rails.logger.info "ClaimID=#{claim.id} RPO=#{claim.education_benefits_claim.region} Form=#{form_type}"
-      validate_session
+      load_user
```

Comment:
I didnt realize that `validate_session` was being called outside of authentication_and_sso_concerns.rb. What's the motivation for a controller action to both:

1. `skip_before_action(:authenticate)`

and,

2. call `validate_session` from within a specific action

? I suppose this question is just for my own understanding and not to do with this PR.

Reply:
On most (all?) of them, they are forms that can be completed by logged-in or logged-out users.  If they were logged-in when completing the form, there will be an `in_progress_forms` record to delete after submission.  Loading the user allows the record deletion to success.  See [`#clear_saved_form`](https://github.com/department-of-veterans-affairs/vets-api/blob/d834efae2725b6a7eeaabb033cdd51d85d8a52a0/app/controllers/application_controller.rb#L42)

The `gi_bill_feedbacks_controller` also includes user data in the form, if present.